builtins/amp-video.md

```diff
@@ -82,9 +83,10 @@ Similar to the `video` tag `controls` attribute - if present, the browser offers
 
 If present, will automatically loop the video back to the start upon reaching the end.
 
-**muted**
+**muted (deprecated)**
```

Comment:
The main issue is that UA's interpret `muted` attribute differently. Here is an example, what do you think this will do:
`<video muted autoplay controls>`?
Safari in iOS 10 interprets this as the video has no audio tracks and does not allow the user to  unmute this. (just try https://jsfiddle.net/q65768nw/ in iOS 10, there is no unmute button, volume controls doesn't do anything. Chrome gives users an unmute UI button with its default state dictated by the `muted` attribute.)

ok crap, so how is the author gonna fix that? They will remove the `muted` attribute. Now what's gonna happen?

oh crap, nothing autoplays on mobile platforms anymore. What to do now?...

At least until Safari in iOS 10 gives users a UI control to unmute a muted auto-playing video, what this attribute does natively is both confusing and can create a situation where users can not unmute a video! So we can't simply document and follow what native `unmute` attribute does.